tests/profiles/googlefonts_test.py

```diff
@@ -547,7 +547,7 @@ def DISABLED_test_check_family_equal_numbers_of_glyphs(mada_ttFonts, cabin_ttFon
 
   # our reference Mada family is bad here with 407 glyphs on most font files
   # except the Black and the Medium, that both have 408 glyphs.
-  asser_results_contain(check(mada_ttFonts),
+  assert_results_contain(check(mada_ttFonts),
                         FAIL, 'glyph-count-diverges',
                         'with fonts that diverge on number of glyphs.')
```

Comment:
please update indentation to keep the lines well aligned ;-)

Reply:
ha...you got me 😸  fixed in https://github.com/googlefonts/fontbakery/pull/2998/commits/4567e9bfc52eb5b0cc55c7cf9f50dae7e69dba2f